templates/config.edn

```diff
@@ -207,145 +242,179 @@
+ ;; - redirect-page?: Redirect to the quick capture page after capturing.
+ ;; - default-page:   The default page to capture to if insert-today? is false.
+  :quick-capture-options
+  {:insert-today? false            ;; Default value: true
+   :redirect-page? false          ;; Default value: false
+   :default-page "quick capture"} ;; Default value: "quick capture" page
 
- ;; Quick capture options
- ;; :quick-capture-options {:insert-today? false :redirect-page? false :default-page "my page"}
 
  ;; File sync options
  ;; Ignore these files when syncing, regexp is supported.
- ;; :file-sync/ignore-files []
-
- ;; dwim (do what I mean) for Enter key when editing.
- ;; Context-awareness of Enter key makes editing more easily
- ; :dwim/settings {
- ;   :admonition&src?  true
- ;   :markup?          false
- ;   :block-ref?       true
- ;   :page-ref?        true
- ;   :properties?      true
- ;   :list?            true
- ; }
-
- ;; Decide the way to escape the special characters in the page title.
+  :file-sync/ignore-files []
+
+ ;; Configure the Enter key behavior for
+ ;; context-aware editing with DWIM (Do What I Mean).
+ ;; context-aware Enter key behavior implies that pressing Enter will
+ ;; have different outcomes based on the context.
+ ;; For instance, pressing Enter within a list generates a new list item,
+ ;; whereas pressing Enter in a block reference opens the referenced block.
+  :dwim/settings
+  {:admonition&src?  true        ;; Default value: true
+   :markup?          false       ;; Default value: false
+   :block-ref?       true        ;; Default value: true
+   :page-ref?        true        ;; Default value: true
+   :properties?      true        ;; Default value: true
+   :list?            true}       ;; Default value: true
+
+ ;; Configure the escaping method for special characters in page titles.
  ;; Warning:
- ;;   This is a dangerous operation. If you want to change the setting,
- ;;   should access the setting `Filename format` and follow the instructions.
- ;;   Or you have to rename all the affected files manually then re-index on all
- ;;   clients after the files are synced. Wrong handling may cause page titles
- ;;   containing special characters to be messy.
- ;; Available values:
- ;;   :file/name-format :triple-lowbar
+ ;;   This is a dangerous operation. To modify the setting,
+ ;;   access the 'Filename format' setting and follow the instructions.
+ ;;   Othwerwise, You may need to manually rename all affected files and
+ ;;   re-index them on all clients after synchronization.
+ ;;   Incorrect handling may result in messy page titles.
+ ;; Available options:
+ ;;   -:triple-lowbar (default)
  ;;     ;use triple underscore `___` for slash `/` in page title
  ;;     ;use Percent-encoding for other invalid characters
- :file/name-format :triple-lowbar
-
- ;; specify the format of the filename for journal files
- ;; :journal/file-name-format "yyyy_MM_dd"
-
- }
+  :file/name-format :triple-lowbar}
```

Comment:
the default value is `legacy` for compatibility reasons. All new graphs should use `:triple-lowbar`

Reply:
One space was added accidentally. It's better to remove it.

And I think it's better add `:legacy` to available options list.